### PR TITLE
Fix to multilingual abstract.

### PR DESCRIPTION
**Pull request by justinlittman**
**NOTE:  Person merging should ensure that any mapping changes are ticketed as a cocina mapping in dor-services-app, e.g. dor-services-app/issues/1251**

## Why was this change made?
Fix to multilingual abstract.

### CODE REVIEW

File: mods_cocina_mappings/mods_to_cocina_abstract.txt [hunk 1]
```diff
@@ -15,11 +15,11 @@
 {
   "note": [
     {
-      "type": "summary"
+      "type": "summary".
```
Comment (jcoyne) [inline mods_cocina_mappings/mods_to_cocina_abstract.txt]: should this be a comma?

**Comment by ndushay** [comment]
@justinlittman has this coding already been done in dor-services-app, or will you make a ticket when this is merged?

File: mods_cocina_mappings/mods_to_cocina_abstract.txt (hunk 1 above)
Comment (justinlittman) [reply mods_cocina_mappings/mods_to_cocina_abstract.txt]: Yes. Fixed.

**Comment by justinlittman** [comment]
Yes, coding has already been done in dor-services-app.

**Review by ndushay — APPROVED** [review]

**Pull request merged**